governance-aepp/src/utils/contract.js

```diff
@@ -0,0 +1,86 @@
+import settings from './settings';
+import registryInterface from '../assets/contracts/RegistryInterface.aes';
+import pollContractSource from '../assets/contracts/Poll.aes';
+import pollIrisContractSource from '../assets/contracts/Poll_Iris.aes';
+import {toRefs} from "vue";
+import {sdk, wallet} from "@/utils/wallet";
+
+const contract = {
+  registry: null
+};
+
+// TODO adjust ACI usage
+
+contract.init = async () => {
+  const {networkId} = toRefs(wallet)
```

Comment:
why not access `wallet.networkId` below?

Reply:
if that works I can try